source/components/cardContainer/filters/filterGroup/filterGroup.directive.ng1.ts

```diff
@@ -54,5 +68,6 @@ export let filterGroup: angular.IComponentOptions = {
 		icon: '=',
 		filterGroup: '=',
 		source: '=',
+		enabled: '<',
```

Comment:
I tend to prefer `enabled` rather than `disabled` because to me it reads a lot easier. If I see something like `!disabled` or `disabled == false` I read it as "not disabled", and especially when chained with a bunch of other boolean expressions I sometimes have to do a double take on it. While something like `!enabled` or `enabled == false` reads "not enabled" which is slightly less double negative. :smile: 

That said, there's something to say for consistency in the code base so I don't mind changing it if that's how it's usually done elsewhere.